Podcast/SearchSectionHeaderView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SearchSectionHeaderTableViewCell.swift
+//  Podcast
+//
+//  Created by Jack Thompson on 4/6/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+protocol SearchTableViewHeaderDelegate: class {
+    func searchTableViewHeaderDidPressViewAllButton(view: SearchSectionHeaderView)
+}
+
+class SearchSectionHeaderView: UIView {
+    
+    let edgePadding: CGFloat = 18
+    var mainLabel: UILabel!
+    var viewAllButton: UIButton!
+    var type: SearchType!
+    weak var delegate: SearchTableViewHeaderDelegate?
+    
+    var mainLabelHeight: CGFloat = 21
+    var viewAllButtonHeight: CGFloat = 18
+    var bottomInset: CGFloat = 12.5
+    
+    init(frame: CGRect, type: SearchType) {
```

Comment:
We actually have a decent amount of headers that look just like this throughout .. it would be a good idea to refactor this out to a generic header view in which the button text and title text can be customized. Doesn't have to be in this PR but would be a great thing to do to refactor and change it throughout our app in a supplemental PR.